### ISSUE TITLE
feature map stride = 4

### ISSUE THREAD

**Issue by ralekz** [ultralytics/ultralytics#22700]
### Search before asking

- [x] I have searched the Ultralytics [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar feature requests.

### Description

Hello, team!
Your neural network claims that your Yolo11 detect models come with "feature map stride parameters" of 32, 16, and 8. How can I also get your detection models with "feature map stride = 4"? Can I download the entire line of detection models with this parameter? Can I change this parameter myself?
I need to predict small details with greater accuracy...

### Use case

Determining the exact year of production of a car based on its appearance using photos or videos

### Additional

YOLO11 n/s/m/l/x detect
GeForce RTX 5070
Dataset of 380,000 cropped from 4K video to 640/640 pixels, carefully labeled with boxes
10% empty frames without labeling added

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @ralekz, thank you for your interest in Ultralytics and for the detailed description of your use case 🚀 This is an automated response to help you get started quickly — an Ultralytics engineer will also review your issue and follow up soon.

Since your question is about model configuration and improving performance on small details (like fine-grained car year recognition), you may find it helpful to review the docs for general usage and training behavior:

- Check the [Docs](https://docs.ultralytics.com/) for an overview of YOLO11 usage and configuration
- See [Python usage examples](https://docs.ultralytics.com/usage/python/) and [CLI usage examples](https://docs.ultralytics.com/usage/cli/) for how to run and customize models in code or from the command line

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training or model behavior ❓ Question (which your small-detail / car-year scenario suggests), please:
- Include key training details (model variant, image size, epochs, batch size, augmentations)
- Share a few representative dataset examples (especially of the smallest objects you care about)
- Add training logs or summaries
- Verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/), as many performance issues are related to data, augmentation, or hyperparameters ⚙️

Join the Ultralytics community where it suits you best:
- For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧
- For in-depth, long-form discussions, see [Discourse](https://community.ultralytics.com/)
- For community-driven threads and sharing results, check out our [Subreddit](https://www.reddit.com/r/Ultralytics/)  

## Upgrade

Before proceeding, please ensure you are on the latest version of `ultralytics` and dependencies. Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a **Python>=3.8** environment with **PyTorch>=1.8** to verify your question or issue is not already addressed in the latest release:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including CUDA/CUDNN, Python and PyTorch preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

An Ultralytics engineer will review your question about feature map stride, YOLO11 detect models, and your small-detail car-year use case and will follow up with more specific guidance soon 🔍🚗

**Comment by Y-T-G** [comment]
You can use this:

https://github.com/Y-T-G/community/blob/main/cfg/detect/yolo11-p2.yaml

**Comment by ralekz** [comment]
Could you please tell me why this YAML only runs on the small model? And how do I run the L, for example?

**Comment by Y-T-G** [comment]
It runs for any size. You just need to add the size to the filename when loading. Don't rename the actual file.

`model = YOLO("yolo11l-p2.yaml")`

**Comment by ralekz** [comment]
> You can use this:
> 
> https://github.com/Y-T-G/community/blob/main/cfg/detect/yolo11-p2.yaml

Hi! You helped me a lot last time, thank you! Do you happen to have a configuration with stride=4 for YOLO26?

**Comment by Y-T-G** [comment]
Not currently. But I might add it later to the Community Configs repo.
